RuntimeLogger 2025-07-12 17:19:09,711 ERROR (main|23) [LogLog.warn():157] log4j:WARN No appenders could be found for logger (io.netty.util.internal.InternalThreadLocalMap).
RuntimeLogger 2025-07-12 17:19:09,712 ERROR (main|23) [LogLog.warn():157] log4j:WARN Please initialize the log4j system properly.
RuntimeLogger 2025-07-12 17:19:09,713 ERROR (main|23) [LogLog.warn():157] log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
RuntimeLogger 2025-07-12 17:19:13,245 WARN (MASTER fe_4bb83929_c370_45e8_812d_0b537bdb87bf(1)|23) [Env.notifyNewFETypeTransfer():2838] notify new FE type transfer: MASTER
RuntimeLogger 2025-07-12 17:19:16,146 WARN (InternalSchemaInitializer|23) [InternalSchemaInitializer.run():79] FE is not ready
RuntimeLogger 2025-07-12 17:19:16,264 WARN (Statistics Table Cleaner|23) [StatisticsCleaner.init():172] Failed to init stats cleaner
java.lang.RuntimeException: Table: `internal.__internal_schema.column_statistics` not exists
	at org.apache.doris.statistics.util.StatisticsUtil.findTable(StatisticsUtil.java:420) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.statistics.StatisticsCleaner.init(StatisticsCleaner.java:167) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.statistics.StatisticsCleaner.clear(StatisticsCleaner.java:82) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.statistics.StatisticsCleaner.runAfterCatalogReady(StatisticsCleaner.java:76) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.util.MasterDaemon.runOneCycle(MasterDaemon.java:58) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.util.Daemon.run(Daemon.java:119) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:21,414 WARN (MASTER fe_4bb83929_c370_45e8_812d_0b537bdb87bf(1)|179) [QeService.start():58] Help module failed. ignore it.
java.io.IOException: Can not find help zip file: help-resource.zip
	at org.apache.doris.qe.help.HelpModule.setUpModule(HelpModule.java:271) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.QeService.start(QeService.java:56) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.DorisFE.start(DorisFE.java:222) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.DorisFE.main(DorisFE.java:97) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:24,496 WARN (mysql-nio-pool-0|179) [StmtExecutor.executeAndSendResult():2075] cancel fragment query_id:f5e5dc2c7d0344c0-a840d5ce16dc5a1d cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
RuntimeLogger 2025-07-12 17:19:24,496 WARN (mysql-nio-pool-0|179) [Coordinator.cancel():1296] Cancel execution of query f5e5dc2c7d0344c0-a840d5ce16dc5a1d, this is a outside invoke, cancelReason Status [errorCode=INTERNAL_ERROR, errorMsg=cancel fragment query_id:f5e5dc2c7d0344c0-a840d5ce16dc5a1d cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.]
RuntimeLogger 2025-07-12 17:19:24,506 WARN (mysql-nio-pool-0|179) [ConnectProcessor.handleQueryException():489] Process one query failed because.
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:281) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:24,511 WARN (mysql-nio-pool-0|179) [ConnectProcessor.handleQuery():254] execute query exception
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:281) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:24,794 WARN (mysql-nio-pool-0|179) [StmtExecutor.executeAndSendResult():2075] cancel fragment query_id:3999131bba4d450b-abcee669c115b3bf cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
RuntimeLogger 2025-07-12 17:19:24,794 WARN (mysql-nio-pool-0|179) [Coordinator.cancel():1296] Cancel execution of query 3999131bba4d450b-abcee669c115b3bf, this is a outside invoke, cancelReason Status [errorCode=INTERNAL_ERROR, errorMsg=cancel fragment query_id:3999131bba4d450b-abcee669c115b3bf cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.]
RuntimeLogger 2025-07-12 17:19:24,795 WARN (mysql-nio-pool-0|179) [ConnectProcessor.handleQueryException():489] Process one query failed because.
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:281) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:24,796 WARN (mysql-nio-pool-0|179) [ConnectProcessor.handleQuery():254] execute query exception
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:281) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:24,814 WARN (mysql-nio-pool-1|179) [StmtExecutor.executeAndSendResult():2075] cancel fragment query_id:93a3430fae864ecd-bb9f57c0df38e96f cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
RuntimeLogger 2025-07-12 17:19:24,814 WARN (mysql-nio-pool-1|179) [Coordinator.cancel():1296] Cancel execution of query 93a3430fae864ecd-bb9f57c0df38e96f, this is a outside invoke, cancelReason Status [errorCode=INTERNAL_ERROR, errorMsg=cancel fragment query_id:93a3430fae864ecd-bb9f57c0df38e96f cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.]
RuntimeLogger 2025-07-12 17:19:24,814 WARN (mysql-nio-pool-1|179) [ConnectProcessor.handleQueryException():489] Process one query failed because.
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:281) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:24,814 WARN (mysql-nio-pool-1|179) [ConnectProcessor.handleQuery():254] execute query exception
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:281) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:24,947 WARN (mysql-nio-pool-2|179) [StmtExecutor.executeAndSendResult():2075] cancel fragment query_id:9bf46cfc872d4c90-bac5479908f68d1a cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]
RuntimeLogger 2025-07-12 17:19:24,947 WARN (mysql-nio-pool-2|179) [Coordinator.cancel():1296] Cancel execution of query 9bf46cfc872d4c90-bac5479908f68d1a, this is a outside invoke, cancelReason Status [errorCode=INTERNAL_ERROR, errorMsg=cancel fragment query_id:9bf46cfc872d4c90-bac5479908f68d1a cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]]
RuntimeLogger 2025-07-12 17:19:24,947 WARN (mysql-nio-pool-2|179) [ConnectProcessor.handleQueryException():489] Process one query failed because.
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:297) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:24,947 WARN (mysql-nio-pool-2|179) [ConnectProcessor.handleQuery():254] execute query exception
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:297) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:25,878 WARN (mysql-nio-pool-2|179) [StmtExecutor.executeAndSendResult():2075] cancel fragment query_id:e3431825016f4b1b-8972c11c4107c7d7 cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]
RuntimeLogger 2025-07-12 17:19:25,878 WARN (mysql-nio-pool-2|179) [Coordinator.cancel():1296] Cancel execution of query e3431825016f4b1b-8972c11c4107c7d7, this is a outside invoke, cancelReason Status [errorCode=INTERNAL_ERROR, errorMsg=cancel fragment query_id:e3431825016f4b1b-8972c11c4107c7d7 cause errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]]
RuntimeLogger 2025-07-12 17:19:25,878 WARN (mysql-nio-pool-2|179) [ConnectProcessor.handleQueryException():489] Process one query failed because.
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:297) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:25,879 WARN (mysql-nio-pool-2|179) [ConnectProcessor.handleQuery():254] execute query exception
org.apache.doris.common.UserException: errCode = 2, detailMessage = No backend available as scan node, please check the status of your backends.[1752340753313: not alive]
	at org.apache.doris.qe.SimpleScheduler.getHost(SimpleScheduler.java:297) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentHosts(Coordinator.java:1761) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.computeFragmentExecParams(Coordinator.java:1342) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.processFragmentAssignmentAndParams(Coordinator.java:617) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.execInternal(Coordinator.java:714) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:680) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeAndSendResult(StmtExecutor.java:1969) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:1927) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.handleQueryWithRetry(StmtExecutor.java:929) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.executeByNereids(StmtExecutor.java:843) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:614) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.queryRetry(StmtExecutor.java:576) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:561) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.executeQuery(ConnectProcessor.java:350) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:250) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.handleQuery(MysqlConnectProcessor.java:207) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.dispatch(MysqlConnectProcessor.java:235) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.qe.MysqlConnectProcessor.processOnce(MysqlConnectProcessor.java:415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
RuntimeLogger 2025-07-12 17:19:26,252 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3223] create table failed [1752340753318] - errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
RuntimeLogger 2025-07-12 17:19:26,255 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3239] drop table
org.apache.doris.common.MetaNotFoundException: errCode = 7, detailMessage = table not found, tableId=1752340753316
	at org.apache.doris.catalog.DatabaseIf.lambda$getTableOrMetaException$1(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrException(DatabaseIf.java:145) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrMetaException(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.dropTable(InternalCatalog.java:1059) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3233) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:26,255 WARN (InternalSchemaInitializer|179) [InternalSchemaInitializer.run():88] Statistics storage initiated failed, will try again later
org.apache.doris.common.DdlException: errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
	at org.apache.doris.system.SystemInfoService.selectBackendIdsForReplicaCreation(SystemInfoService.java:507) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTablets(InternalCatalog.java:3415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createPartitionWithIndices(InternalCatalog.java:2146) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3076) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:26,380 WARN (heartbeat-mgr-pool-1|179) [HeartbeatMgr$BackendHeartbeatHandler.pingOnce():334] backend heartbeat got exception
org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused
	at org.apache.thrift.transport.TSocket.open(TSocket.java:255) ~[libthrift-0.16.0.jar:0.16.0]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:143) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:126) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.commons.pool2.BaseKeyedPooledObjectFactory.makeObject(BaseKeyedPooledObjectFactory.java:62) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.create(GenericKeyedObjectPool.java:1012) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:356) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:277) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.doris.common.GenericPool.borrowObject(GenericPool.java:95) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.pingOnce(HeartbeatMgr.java:270) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:241) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:230) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:672) ~[?:?]
	at sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at org.apache.thrift.transport.TSocket.open(TSocket.java:250) ~[libthrift-0.16.0.jar:0.16.0]
	... 14 more
RuntimeLogger 2025-07-12 17:19:26,382 WARN (heartbeat-mgr-pool-1|179) [HeartbeatMgr$BackendHeartbeatHandler.pingOnce():334] backend heartbeat got exception
org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused
	at org.apache.thrift.transport.TSocket.open(TSocket.java:255) ~[libthrift-0.16.0.jar:0.16.0]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:143) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:126) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.commons.pool2.BaseKeyedPooledObjectFactory.makeObject(BaseKeyedPooledObjectFactory.java:62) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.create(GenericKeyedObjectPool.java:1012) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:356) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:277) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.doris.common.GenericPool.borrowObject(GenericPool.java:95) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.pingOnce(HeartbeatMgr.java:270) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:244) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:230) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:672) ~[?:?]
	at sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at org.apache.thrift.transport.TSocket.open(TSocket.java:250) ~[libthrift-0.16.0.jar:0.16.0]
	... 14 more
RuntimeLogger 2025-07-12 17:19:26,383 WARN (heartbeat mgr|179) [HeartbeatMgr.runAfterCatalogReady():158] get bad heartbeat response: type: BACKEND, status: BAD, msg: java.net.ConnectException: Connection refused, beId: 1752340753313, beHost: 172.20.80.3, bePort: 0, httpPort: 0, brpcPort: 0, arrowFlightSqlPort: 0
RuntimeLogger 2025-07-12 17:19:36,280 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3223] create table failed [1752340753336] - errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
RuntimeLogger 2025-07-12 17:19:36,281 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3239] drop table
org.apache.doris.common.MetaNotFoundException: errCode = 7, detailMessage = table not found, tableId=1752340753334
	at org.apache.doris.catalog.DatabaseIf.lambda$getTableOrMetaException$1(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrException(DatabaseIf.java:145) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrMetaException(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.dropTable(InternalCatalog.java:1059) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3233) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:36,281 WARN (InternalSchemaInitializer|179) [InternalSchemaInitializer.run():88] Statistics storage initiated failed, will try again later
org.apache.doris.common.DdlException: errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
	at org.apache.doris.system.SystemInfoService.selectBackendIdsForReplicaCreation(SystemInfoService.java:507) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTablets(InternalCatalog.java:3415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createPartitionWithIndices(InternalCatalog.java:2146) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3076) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:36,406 WARN (heartbeat-mgr-pool-3|179) [HeartbeatMgr$BackendHeartbeatHandler.pingOnce():334] backend heartbeat got exception
org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused
	at org.apache.thrift.transport.TSocket.open(TSocket.java:255) ~[libthrift-0.16.0.jar:0.16.0]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:143) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:126) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.commons.pool2.BaseKeyedPooledObjectFactory.makeObject(BaseKeyedPooledObjectFactory.java:62) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.create(GenericKeyedObjectPool.java:1012) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:356) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:277) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.doris.common.GenericPool.borrowObject(GenericPool.java:95) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.pingOnce(HeartbeatMgr.java:270) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:241) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:230) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:672) ~[?:?]
	at sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at org.apache.thrift.transport.TSocket.open(TSocket.java:250) ~[libthrift-0.16.0.jar:0.16.0]
	... 14 more
RuntimeLogger 2025-07-12 17:19:36,407 WARN (heartbeat-mgr-pool-3|179) [HeartbeatMgr$BackendHeartbeatHandler.pingOnce():334] backend heartbeat got exception
org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused
	at org.apache.thrift.transport.TSocket.open(TSocket.java:255) ~[libthrift-0.16.0.jar:0.16.0]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:143) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:126) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.commons.pool2.BaseKeyedPooledObjectFactory.makeObject(BaseKeyedPooledObjectFactory.java:62) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.create(GenericKeyedObjectPool.java:1012) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:356) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:277) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.doris.common.GenericPool.borrowObject(GenericPool.java:95) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.pingOnce(HeartbeatMgr.java:270) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:244) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:230) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:672) ~[?:?]
	at sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at org.apache.thrift.transport.TSocket.open(TSocket.java:250) ~[libthrift-0.16.0.jar:0.16.0]
	... 14 more
RuntimeLogger 2025-07-12 17:19:36,407 WARN (heartbeat mgr|179) [HeartbeatMgr.runAfterCatalogReady():158] get bad heartbeat response: type: BACKEND, status: BAD, msg: java.net.ConnectException: Connection refused, beId: 1752340753313, beHost: 172.20.80.3, bePort: 0, httpPort: 0, brpcPort: 0, arrowFlightSqlPort: 0
RuntimeLogger 2025-07-12 17:19:47,230 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3223] create table failed [1752340753354] - errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
RuntimeLogger 2025-07-12 17:19:47,230 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3239] drop table
org.apache.doris.common.MetaNotFoundException: errCode = 7, detailMessage = table not found, tableId=1752340753352
	at org.apache.doris.catalog.DatabaseIf.lambda$getTableOrMetaException$1(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrException(DatabaseIf.java:145) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrMetaException(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.dropTable(InternalCatalog.java:1059) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3233) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:47,230 WARN (InternalSchemaInitializer|179) [InternalSchemaInitializer.run():88] Statistics storage initiated failed, will try again later
org.apache.doris.common.DdlException: errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
	at org.apache.doris.system.SystemInfoService.selectBackendIdsForReplicaCreation(SystemInfoService.java:507) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTablets(InternalCatalog.java:3415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createPartitionWithIndices(InternalCatalog.java:2146) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3076) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:47,354 WARN (heartbeat-mgr-pool-5|179) [HeartbeatMgr$BackendHeartbeatHandler.pingOnce():334] backend heartbeat got exception
org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused
	at org.apache.thrift.transport.TSocket.open(TSocket.java:255) ~[libthrift-0.16.0.jar:0.16.0]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:143) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:126) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.commons.pool2.BaseKeyedPooledObjectFactory.makeObject(BaseKeyedPooledObjectFactory.java:62) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.create(GenericKeyedObjectPool.java:1012) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:356) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:277) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.doris.common.GenericPool.borrowObject(GenericPool.java:95) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.pingOnce(HeartbeatMgr.java:270) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:241) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:230) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:672) ~[?:?]
	at sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at org.apache.thrift.transport.TSocket.open(TSocket.java:250) ~[libthrift-0.16.0.jar:0.16.0]
	... 14 more
RuntimeLogger 2025-07-12 17:19:47,356 WARN (heartbeat-mgr-pool-5|179) [HeartbeatMgr$BackendHeartbeatHandler.pingOnce():334] backend heartbeat got exception
org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused
	at org.apache.thrift.transport.TSocket.open(TSocket.java:255) ~[libthrift-0.16.0.jar:0.16.0]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:143) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.common.GenericPool$ThriftClientFactory.create(GenericPool.java:126) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.commons.pool2.BaseKeyedPooledObjectFactory.makeObject(BaseKeyedPooledObjectFactory.java:62) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.create(GenericKeyedObjectPool.java:1012) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:356) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.commons.pool2.impl.GenericKeyedObjectPool.borrowObject(GenericKeyedObjectPool.java:277) ~[commons-pool2-2.2.jar:2.2]
	at org.apache.doris.common.GenericPool.borrowObject(GenericPool.java:95) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.pingOnce(HeartbeatMgr.java:270) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:244) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.system.HeartbeatMgr$BackendHeartbeatHandler.call(HeartbeatMgr.java:230) ~[doris-fe.jar:1.2-SNAPSHOT]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:672) ~[?:?]
	at sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542) ~[?:?]
	at sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597) ~[?:?]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327) ~[?:?]
	at java.net.Socket.connect(Socket.java:633) ~[?:?]
	at org.apache.thrift.transport.TSocket.open(TSocket.java:250) ~[libthrift-0.16.0.jar:0.16.0]
	... 14 more
RuntimeLogger 2025-07-12 17:19:47,358 WARN (heartbeat mgr|179) [HeartbeatMgr.runAfterCatalogReady():158] get bad heartbeat response: type: BACKEND, status: BAD, msg: java.net.ConnectException: Connection refused, beId: 1752340753313, beHost: 172.20.80.3, bePort: 0, httpPort: 0, brpcPort: 0, arrowFlightSqlPort: 0
RuntimeLogger 2025-07-12 17:19:57,240 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3223] create table failed [1752340753372] - errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
RuntimeLogger 2025-07-12 17:19:57,240 WARN (InternalSchemaInitializer|179) [InternalCatalog.createOlapTable():3239] drop table
org.apache.doris.common.MetaNotFoundException: errCode = 7, detailMessage = table not found, tableId=1752340753370
	at org.apache.doris.catalog.DatabaseIf.lambda$getTableOrMetaException$1(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrException(DatabaseIf.java:145) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.DatabaseIf.getTableOrMetaException(DatabaseIf.java:156) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.dropTable(InternalCatalog.java:1059) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3233) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:19:57,240 WARN (InternalSchemaInitializer|179) [InternalSchemaInitializer.run():88] Statistics storage initiated failed, will try again later
org.apache.doris.common.DdlException: errCode = 2, detailMessage = replication num should be less than the number of available backends. replication num is 1, available backend num is 0
	at org.apache.doris.system.SystemInfoService.selectBackendIdsForReplicaCreation(SystemInfoService.java:507) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTablets(InternalCatalog.java:3415) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createPartitionWithIndices(InternalCatalog.java:2146) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createOlapTable(InternalCatalog.java:3076) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.datasource.InternalCatalog.createTable(InternalCatalog.java:1257) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.createTbl(InternalSchemaInitializer.java:245) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.catalog.InternalSchemaInitializer.run(InternalSchemaInitializer.java:86) ~[doris-fe.jar:1.2-SNAPSHOT]
RuntimeLogger 2025-07-12 17:20:05,135 WARN (qtp2138645808-192|179) [RestApiExceptionHandler.unauthorizedHandler():40] unauthorized exception
org.apache.doris.httpv2.exception.UnauthorizedException: Access denied for user 'engineer@172.20.80.1' (using password: YES)
	at org.apache.doris.httpv2.controller.BaseController.checkPassword(BaseController.java:261) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.httpv2.controller.BaseController.checkWithCookie(BaseController.java:71) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.httpv2.controller.BaseController.checkAuthWithCookie(BaseController.java:61) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.apache.doris.httpv2.controller.LoginController.login(LoginController.java:35) ~[doris-fe.jar:1.2-SNAPSHOT]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:205) ~[spring-web-5.3.39.jar:5.3.39]
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:150) ~[spring-web-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:117) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:903) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:809) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1072) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:965) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707) ~[javax.servlet-api-3.1.0.jar:3.1.0]
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883) ~[spring-webmvc-5.3.39.jar:5.3.39]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) ~[javax.servlet-api-3.1.0.jar:3.1.0]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1656) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.websocket.server.WebSocketUpgradeFilter.doFilter(WebSocketUpgradeFilter.java:292) ~[websocket-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.apache.doris.httpv2.interceptor.ServletTraceIterceptor.doFilter(ServletTraceIterceptor.java:55) ~[doris-fe.jar:1.2-SNAPSHOT]
	at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:201) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.3.39.jar:5.3.39]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.39.jar:5.3.39]
	at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93) ~[spring-web-5.3.39.jar:5.3.39]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.39.jar:5.3.39]
	at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) ~[spring-web-5.3.39.jar:5.3.39]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.39.jar:5.3.39]
	at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:193) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1626) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:552) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:600) ~[jetty-security-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1440) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:505) ~[jetty-servlet-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1355) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.Server.handle(Server.java:516) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:487) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:732) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:479) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277) ~[jetty-server-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311) ~[jetty-io-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105) ~[jetty-io-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104) ~[jetty-io-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:338) ~[jetty-util-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:315) ~[jetty-util-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173) ~[jetty-util-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:137) ~[jetty-util-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883) ~[jetty-util-9.4.57.v20241219.jar:9.4.57.v20241219]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034) ~[jetty-util-9.4.57.v20241219.jar:9.4.57.v20241219]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
